app/color_number

```diff
@@ -0,0 +1,64 @@
+const { MajorColorNames, MinorColorNames, ColorPair }= require ("./color");
+
+function getColorFromPairNumber(pairNumber)
+{
+	let minorSize = MajorColorNames.length;
+	let majorSize = MinorColorNames.length;
+	
+	if (pairNumber < 1 || pairNumber > minorSize * majorSize)
```

Comment:
Nice range check :)
But how would you prove that it's working? What would it take to write a test for this?